### PR TITLE
Add 'Extend duration' option for breakout rooms

### PR DESCRIPTION
**Pull request by gustavotrott**
This PR implements the feature requested in #9435 

- Moderators can extend breakout rooms time before it ends

extend_time

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/breakout-room/component.jsx [hunk 1]
```diff
@@ -155,6 +173,18 @@ class BreakoutRoom extends PureComponent {
     this.setState({ joinedAudioOnly: false });
   }
 
+  changeExtendTime(event) {
+    this.setState({ extendTime: Number.parseInt(event.target.value, 10) || '' });
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/breakout-room/component.jsx:177]:
```suggestion
    this.setState({ extendTime: Number.parseInt(event.target.value, 10) || 0 });
```

**Review by antobinary — COMMENTED** [review]
I do not spot the code to handle the addition of `-5` minutes... Looking at the code, that would indeed decrease the time remaining. Pls comment

**Comment by gustavotrott** [comment]
> 
> 
> I do not spot the code to handle the addition of `-5` minutes... Looking at the code, that would indeed decrease the time remaining. Pls comment

I just included some validations to avoid this situation!

**Comment by gustavotrott** [comment]
Included the validation related with parent meeting remaining time.
Now it's ready for review @antobinary!

File: bigbluebutton-html5/imports/ui/components/breakout-room/component.jsx [hunk 2]
```diff
@@ -334,14 +374,87 @@ class BreakoutRoom extends PureComponent {
...
-          messageDuration={intlMessages.breakoutDuration}
-          breakoutRoom={breakoutRooms[0]}
-        />
-      </span>
+      <div className={styles.durationContainer}>
+        { amIModerator && visibleExtendTimeForm ? (
+          <div className={styles.extendTimeContainer}>
+            <label
+              htmlFor="inputExtendTimeSelector"
+              className={cx(styles.label, styles.labelSmall)}
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/breakout-room/component.jsx:387]: Screenshot from 2021-06-11 16-14-40
Screenshot from 2021-06-11 16-14-33

**Review by antobinary — APPROVED** [review]
Worked well locally.
The missing CSS classes can be adjusted later

**Pull request merged**